duckduckgo/iOS
# Fix for label not translating on screen "Unprotected Sites"

Task/Issue URL: https://app.asana.com/0/414709148257752/1202166947222438/f
Tech Design URL:
CC:

**Description**:
Text "These sites will not be enhanced by Privacy Protection" has been translated but translation was not being applied

**Steps to test this PR**:
1. Set your phone language to any language the app supports that isn't English 
2. Go to Settings -> Unprotected Sites
3. Confirm that the label "These sites will not be enhanced by Privacy Protection" is translated



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
